client/src/index.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import $ from 'jquery';
+
+
+class App extends React.Component {
+  constructor(props){
+    super(props);
+    this.state = {
+      stocks: [{
+        name: '',
+        rating: 0,
+        ratingBlurb: '',
+        price: 0,
+        priceChange: 0,
+      }],
+    }
+    this.handleGet.bind(this);
+  }
+  componentDidMount() {
+    this.handleGet();  
+  }
+
+  handleGet() {
+    $.get('http://localhost:3007/api', (data) => console.log(data)
```

Comment:
From the project milestones on Learn: "Do not use any jQuery UI components: instead opt for using React Libraries for subordinate UI components". I think you'll need to try this with something else, but I might also be confused as to what they mean by jQuery UI components